### DISCUSSION TITLE
Wow, I must love Rebound 😭

### DISCUSSION THREAD

**Discussion by Lamparter** [IviriusCommunity/Rebound#62] [Show and tell]
Rebound has taken over FluentHub (the only app ever to take it over!) from my top repositories 😮 

![image](https://github.com/user-attachments/assets/7eac9bbf-598f-490e-8080-124984cff6d4)

I also took over @IviriusMain in commits on _his own_ project 😭 

![image](https://github.com/user-attachments/assets/18d29eab-8897-4d8e-8778-65be44328172)

**Comment by Lamparter** [comment]
I just realised that if you expand that list of top repositories it shows you every repo you every contributed to....

![image](https://github.com/user-attachments/assets/d995105f-4193-467e-927c-9c2332982645)

**Comment by IviriusMain** [comment]
Blud became me
